solutions/banking/watson-scripts/assistant-create.sh

```diff
@@ -2,16 +2,8 @@
 
 set -e
 
-WATSON_ASSISTANT_URL=$1
 project_name="gen-ai-rag-sample-app-assistant"
```

Comment:
Would be better if this is a parameter / env var. Updating the script later when resources are already deployed could cause some nasty bahavior.